python/sglang/srt/layers/moe/topk.py

```diff
@@ -231,6 +230,8 @@ def biased_grouped_topk(
             num_expert_group,
             topk_group,
             topk,
+            n_share_experts_fusion,
+            2.5,  # DeepSeek V3/R1 routed_scaling_factor
```

Comment:
can we avoid this constant and load it from config.json?

Reply:
Make sense, done.